src/orqviz/geometric.py

```diff
@@ -1,12 +1,12 @@
-from typing import Optional, Tuple
+from typing import Optional, Tuple, Union
 
 import numpy as np
 from scipy.interpolate import interp1d
 
 from .aliases import ArrayOfParameterVectors, ParameterVector
 
 
-def get_random_normal_vector(dimension: int) -> ParameterVector:
+def get_random_normal_vector(dimension: Union[int, Tuple]) -> ParameterVector:
```

Comment:
I don't see why would you allow for tuple here?
It looks like tuple would change the behaviour to `get_random_normal_array` instead of `vector`?

Reply:
The tuple `(n,m)` would indicate that the parameter vector is `n,m` dimensional.

Reply:
Just like `numpy` accepts `size=n` or `size=(n, m)`